src/js/main.js

```diff
@@ -1,6 +1,78 @@
-import {animatedScrollTo} from './es6-scroll-to.js';
+console.log("ES6");
+const contactUs = {
 
-document.querySelector('.title').innerHTML = 'Great success!'
-document.querySelector('.text').innerHTML = 'Javascript is working!'
+  button: document.getElementsByClassName("imageText__button"),
+  ctaSection: document.querySelectorAll('.contactUs'),
+  opened: "contactUs--opened",
+  closed: "contactUs--closed",
 
-animatedScrollTo(500);
\ No newline at end of file
+  init() {
+    this.toggleBox();
+  },
+
+  slideDown(elementToSlideOpen) {
+    elementToSlideOpen.classList.remove(this.closed);
+    elementToSlideOpen.classList.add(this.opened);
+  },
+
+  slideUp(elem) {
+    elem.classList.remove(this.opened);
+    elem.classList.add(this.closed);
+  },
+
+  toggleBox() {
+    let self = this;
```

Comment:
Caching the context is not necessary in ES6 because of the way arrow functions handle scope as opposed to how classic ES5 functions do it.